src/cache/precomputed_quantities.jl

```diff
@@ -64,6 +64,14 @@ function precomputed_quantities(Y, atmos)
     sedimentation_quantities =
         atmos.moisture_model isa NonEquilMoistModel ?
         (; ᶜwₗ = similar(Y.c, FT), ᶜwᵢ = similar(Y.c, FT)) : (;)
+    moisture_sgs_quantities =
+        atmos.moisture_model isa NonEquilMoistModel ?
+        (;
+            ᶜq_liq⁰ = similar(Y.c, FT),
+            ᶜq_ice⁰ = similar(Y.c, FT),
```

Comment:
```suggestion
```
Can we instead compute these on the fly?

Reply:
They're also already available in the cache as `p.ᶜspecific.q_liq` and `p.ᶜspecific.q_ice`